src/set_theory/surreal.lean

```diff
@@ -198,6 +202,166 @@ theorem numeric_nat : Π (n : ℕ), numeric n
 theorem numeric_omega : numeric omega :=
 ⟨by rintros ⟨⟩ ⟨⟩, λ i, numeric_nat i.down, by rintros ⟨⟩⟩
 
+theorem zero_lt_one : (0 : pgame) < 1 :=
+begin
+  rw lt_def,
+  left,
+  use ⟨punit.star, by split; rintro ⟨ ⟩⟩,
+end
+
+/-- The pre-game `half` is defined as `{0 | 1}`. -/
+def half : pgame := ⟨punit, punit, 0, 1⟩
+
+@[simp] lemma half_move_left : half.move_left punit.star = 0 := rfl
+
+@[simp] lemma half_move_right : half.move_right punit.star = 1 := rfl
+
+/-- The pre-game `half` is numeric. -/
+theorem numeric_half : numeric half :=
+begin
+  split,
+  { rintros ⟨ ⟩ ⟨ ⟩,
+    exact zero_lt_one },
+  split; rintro ⟨ ⟩,
+  { exact numeric_zero },
+  { exact numeric_one }
+end
+
+theorem zero_lt_half : 0 < half :=
+begin
+  rw lt_def,
```

Comment:
It's not working without the `rw`. I think this is happening because `lt_def` is not the direct definition of `lt` but rather a theorem. The original definition of `lt` is `le_lt.2`.